fastchat/model/model_adapter.py

```diff
@@ -504,10 +507,43 @@ def match(self, model_path: str):
 
     def get_default_conv_template(self, model_path: str) -> Conversation:
         return get_conv_template("h2ogpt")
+    
+class LoraAdapter(BaseAdapter):
+    def match(self, model_path: str):
+        # check if has adapter config in model_path
+        res = os.path.exists(os.path.join(model_path, "adapter_config.json"))
```

Comment:
Would this work for someone loading an adapter model from the HuggingFace hub? From my understanding, downloading happens in `load_model` so the adapter config may be remote at this step.